code/game/objects/structures/fluff.dm

```diff
@@ -49,3 +49,9 @@
 	desc = "Although comfortable, this sleeper won't function as anything but a bed ever again."
 	icon = 'icons/obj/lavaland/spawners.dmi'
 	icon_state = "cryostasis_sleeper_open"
+
+/obj/structure/fluff/broken_flooring
+	name = "broken tiling"
+	desc = "A segment of broken flooring"
+	icon = 'icons/obj/brokentiling.dmi'
```

Comment:
These sprites don't align properly with most of the floor/plating sprites we have, especially the 'plating' sprite when set to any direction other than SOUTH.

the 'corner' and 'side' floor tiles also don't use the same floor tile sprite as we do(and have the same rotation problem as the 'plating' sprite)